mkdocs.yml

```diff
@@ -41,13 +41,12 @@ markdown_extensions:
   - pymdownx.emoji:
       emoji_index: !!python/name:materialx.emoji.twemoji
       emoji_generator: !!python/name:materialx.emoji.to_svg
-  - pymdownx.superfences # Indended code blocks, e.g. within lists, admonition, details or tabbed blocks
-  - pymdownx.tabbed # Tabbed content blocks
-  - pymdownx.tasklist: # Task list support
-      custom_checkbox: true # Custom icons
+  - pymdownx.keys # Keyboard key symbols
   - pymdownx.magiclink: # Auto-links non angle bracket URLs and email addresses and allows to shorten and visualise Git repo links
       repo_url_shortener: true
       repo_url_shorthand: true
+  - pymdownx.superfences # Indended code blocks, e.g. within lists, admonition, details or tabbed blocks
+  - pymdownx.tabbed # Tabbed content blocks
```

Comment:
Strange, @fpetru do you have any idea why the above changes broke MkDocs parsing? https://github.com/MichaIng/DietPi-Docs/runs/1703693561?check_suite_focus=true
(The two little indentation alignments were attempts by me to fix the issue.)
YAML validators report correct syntax, aside of the `!!python/name:` values which are special MkDocs syntax to run Python code, but those were there before.

I don't even understand the error message as there is no block starting at the first line, at least if you do not count a whole YAML file as block:
```
MkDocs encountered as error parsing the configuration file: while parsing a block mapping
  in "/home/runner/work/DietPi-Docs/DietPi-Docs/mkdocs.yml", line 1, column 1
expected <block end>, but found '<block sequence start>'
  in "/home/runner/work/DietPi-Docs/DietPi-Docs/mkdocs.yml", line 49, column 3
```

Reply:
Interesting, I use the exact same file (verified with `diff`) on our server and it builds the site without any issue 😕. Python version and all modules match, strange. Let's see if the complete branch builds fine.

Reply:
That solved it: https://github.com/MichaIng/DietPi-Docs/pull/314/commits/3b6587b1ca8f1a50eaf7b797026b88d28085baa6
Mysterious, as locally it doesn't matter. I worry now that something could be parsed wrong, as correct syntax (the way it looks) produces an error.

Reply:
Mystery solved, all working as expected now.